util/git/client.go

```diff
@@ -515,16 +515,16 @@ func (m *nativeGitClient) runCmd(args ...string) (string, error) {
 
 // runCredentialedCmd is a convenience function to run a git command with username/password credentials
 // nolint:unparam
-func (m *nativeGitClient) runCredentialedCmd(command string, args ...string) error {
+func (m *nativeGitClient) runCredentialedCmd(command string, args ...string) (string, error) {
```

Comment:
May I ask the reason for this change? I have not seen the `string` return value being used anywhere in this change.

Reply:
My bad!! I missed reverting this change.